DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -120,7 +120,10 @@ private void OnInitialized(DevCycleEventArgs e)
 
         private string GetConfigUrl()
         {
-            return localOptions.CdnSlug != "" ? localOptions.CdnSlug : $"/config/v1/server/{sdkKey}.json";
+            if(localOptions.CdnSlug.Contains("v1")){
+                logger.LogWarning("Using v1 config endpoint. Please update to v2.");
```

Comment:
@JamieSinn does this sound alright? It will be part of the other communications regarding this release as well

Reply:
I would just add a blanket warning during initialization of the actual client to say "You're overriding the config slug - please make sure to update your path according to config CDN updates. See more <docs link"